### PR TITLE
Update the extension details

### PR DESCRIPTION
**Pull request by MingjiaLiu1995**

### CODE REVIEW

**Comment by MingjiaLiu1995** [comment]
Hi @AlexJerabek , could you please help us review this readme file? The content will show in the VSC marketplace - Office add-in development kit extension page. Thanks!
image

**Review by hermanwenhe — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,39 +2,32 @@
...
+* Ensure you've installed the Microsoft Office Add-ins Development Kit for Visual Studio Code.
+* Select the Office Add-ins Development Kit icon in the Visual Studio Code **Activity Bar** to open the extension.
+* Select **Create a New Add-in**.
+* In the drop down that opens, select an Office app that you want to build the add-in for.
 * Select an add-in template from the list of available templates.
 * Select JavaScript/TypeScript as the programming language.
-* Choose a location where your new add-in will be created in a new folder.
-* Type a name for your project and hit Enter.
+* In the **Workspace folder** dialog that opens, select the folder where you want to create the project.
+* Give a name to the project (with no spaces) when prompted. Office Add-ins Development Kit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.
```
Comment (AlexJerabek) [inline README.md:30]: Why no spaces?

Comment (AlexJerabek) [inline README.md:30]: I think it'll emphasize the behavior better if you italicize less of the line.
```suggestion
* Give a name to the project (with no spaces) when prompted. Office Add-ins Development Kit will create the project with basic files and scaffolding. It will then open the project in a *new* Visual Studio Code window. You are free to close the original Visual Studio Code window.

```

File: README.md [hunk 2]
```diff
@@ -44,31 +37,34 @@ An Office add-in includes two basic components: an XML manifest file, and your o
...
 - The `./src/taskpane/taskpane.css` file contains the CSS that's applied to content in the task pane.
-- The `./src/taskpane/taskpane.js` file contains the Office JavaScript API code that facilitates interaction between the task pane and the Word application.
+- The `./src/taskpane/taskpane.js` file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office application.
 
 ### Preview your add-in in Office apps
 
 To understand how the add-in will work in Office apps, you can use the Office Add-ins Development Kit to easily run and debug your Office add-in in your local dev environment.
-<br><img src=./assets/Dev_Kit_Preview.gif>
-
+<br><img src=./assets/Dev_Kit_Preview.gif alt="Preview an add-in using Office Add-ins Development Kit"/>
```
Comment (AlexJerabek) [inline README.md:48]: The alt text needs to fully describe what is happening to a person who can't see the GIF.
```suggestion
<br><img src=./assets/Dev_Kit_Preview.gif alt="A user selecting the 'Preview' option from the Office Add-ins Development Kit task pane, then choosing the 'Edge Desktop' option to run the sample."/>

```

File: README.md [hunk 3]
```diff
@@ -44,31 +37,34 @@ An Office add-in includes two basic components: an XML manifest file, and your o
...
 
 #### Preview Your Office Add-in (F5)
 
-Select `Preview Your Office Add-in(F5)` on the side panel to start running and debugging the add-in code. The Office Add-ins Development Kit will firstly chenck and install dependencies, and launch A Word/Excel/PowerPoint app to sideload the add-in sample.
-- You can also start debugging by hitting the `F5` key or running `npm run start` command in the terminal.
-- To debug on Office on the web, go to [Sideload Office Add-ins to Office on the web](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing)
-- To debug in Desktop (Edge Legacy), go to [Debug Edge Legacy Webview](https://learn.microsoft.com/office/dev/add-ins/testing/debug-add-ins-using-devtools-edge-legacy)
+Select **Preview Your Office Add-in(F5)** to launch the add-in and debug the code. In the drop down menu, select the option **Desktop (Edge Chromium)**.
```
Comment (AlexJerabek) [inline README.md:52]: "dropdown" is one word in this context. Also, if you can refer to the actual name of this menu, instead of the control type, that would be better.

Also, the option seems to be named "Edge Desktop (Edge Chromium)".
```suggestion
Select **Preview Your Office Add-in(F5)** to launch the add-in and debug the code. In the dropdown menu, select the option **Edge Desktop (Edge Chromium)**.

```

File: README.md [hunk 4]
```diff
@@ -44,31 +37,34 @@ An Office add-in includes two basic components: an XML manifest file, and your o
...
 
-#### Stop Previewing Your Office Add-in
+The extension then checks that the prerequisites are met before debugging starts. Check the terminal for detailed information if there are issues with your environment. After this process, the Office desktop application launches and sideloads the add-in.
```
Comment (AlexJerabek) [inline README.md:54]: Avoid repeating the word "check".
```suggestion
The extension then checks that the prerequisites are met before debugging starts. The terminal will display detailed information if there are issues with your environment. After this process, the Office desktop application launches and sideloads the add-in.
```

File: README.md [hunk 5]
```diff
@@ -44,31 +37,34 @@ An Office add-in includes two basic components: an XML manifest file, and your o
...
+
+You can also start debugging by hitting the `F5` key or running `npm run start` command in the terminal.
```
Comment (AlexJerabek) [inline README.md:56]: Avoid the word "hitting" in favor of "pressing". It will translate better.
```suggestion
You can also start debugging by pressing the `F5` key or running the `npm run start` command in the terminal.
```

**Comment by lindalu-MSFT** [comment]
Is there a preview link I can review?

File: README.md [hunk 6]
```diff
@@ -16,20 +16,15 @@ Verify you have the right prerequisites for building Office add-ins and install
 
 - **[Node.js](https://nodejs.org) 16, 18, or 20 (18 is preferred) and [npm](https://www.npmjs.com/get-npm).** To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
```
[the remark this answers is not in the document]
Comment (hermanwenhe) [reply README.md]: Node should be installed by developer themselves. We will run a checker in Dev Kit to inform them if Node.js is not installed.

**Comment by hermanwenhe** [comment]
> Is there a preview link I can review?

Use this: https://github.com/OfficeDev/OfficeAddinDevKit/tree/MingjiaLiu1995-patch-1

File: README.md [hunk 7]
```diff
@@ -2,12 +2,12 @@
 ## What is Office Add-ins Development Kit?
```
[the remark this answers is not in the document]
Comment (hermanwenhe) [reply README.md:2]: Typo fixed by Copilot. Pls check.

**Comment by lindalu-MSFT** [comment]
[like]  Linda Lu Cannon reacted to your message:
________________________________
